modular_citadel/code/modules/reagents/objects/items.dm

```diff
@@ -0,0 +1,105 @@
+/obj/item/FermiChem/pHbooklet
+    name = "pH indicator booklet"
+    desc = "A booklet containing paper soaked in universal indicator."
+    icon_state = "pHbooklet"
+    icon = 'modular_citadel/icons/obj/FermiChem.dmi'
+    item_flags = NOBLUDGEON
+    var/numberOfPages = 50
+    resistance_flags = FLAMMABLE
+    w_class = WEIGHT_CLASS_TINY
+    //set flammable somehow
+
+//A little janky with pockets
+/obj/item/FermiChem/pHbooklet/attack_hand(mob/user)
+	if(user.get_held_index_of_item(src))
+		if(numberOfPages >= 1)
+			var/obj/item/FermiChem/pHpaper/P = new /obj/item/FermiChem/pHpaper
+			P.add_fingerprint(user)
+			P.forceMove(user.loc)
+			user.put_in_active_hand(P)
+			to_chat(user, "<span class='notice'>You take [P] out of \the [src].</span>")
+			numberOfPages--
+			playsound(user.loc, 'sound/items/poster_ripped.ogg', 50, 1)
+			add_fingerprint(user)
+			if(numberOfPages == 0)
+				icon_state = "pHbookletEmpty"
+			return
+		else
+			to_chat(user, "<span class='warning'>[src] is empty!</span>")
+			add_fingerprint(user)
+			return
+    ..()
```

Comment:
pretty sure the indenting is off on this thing but also it has to do . = ..() and check for the component signal for "no attack hand" unless there' sa really good reason not to.

Reply:
Fixed